spec.bs

```diff
@@ -205,6 +202,24 @@ To <dfn>determine eligibility for an associated site</dfn> given a [=site=] |sit
 3. If |index| is greater than or equal to the [=limit for associated sites=], return false.
 4. Return true.
 
+<h2 id="storage-access-integration">Integration with the Storage Access API</h2>
+
+Modify {{Document/requestStorageAccess()}} to insert the following steps before step 13.5 (i.e. before [=requesting permission to use=]):
```

Comment:
Ideally, the other spec would provide a hook (a `<dfn>` term you could refer to) that you'd just fill in with the steps here, instead of needing to monkeypatch. I'm actually not sure if it'd be better for the hook to come from `requestStorageAccess()` or from [request permission to use](https://w3c.github.io/permissions/#dfn-request-permission-to-use): request-permission-to-use would be really general and would match the idea of each browser making some of its own choices about permissions.

Reply:
Yeah on SAA side we actually think that we'd like to offload more of the implementation-defined behavior to permissions, as part of that I think it makes sense to define an implicitly granted/implicitly denied mechanism in the permissions spec instead.